blackbox/docs/config/node.rst

```diff
@@ -271,10 +271,10 @@ CPU
   Docker, the number of processors may be guessed incorrectly. If this happens,
   you can manually configure the number of processors using this setting.
 
-  You can also use this setting to manually constrain the number of CPUs made
-  available to CrateDB. You might want to do this if you're running CrateDB in
-  a multitenant setup (i.e. more than one CrateDB node running on the same
-  hardware).
+  You can also use this setting to manually constrain the number of CPUs CrateDB
+  will use to adjust internal components like sizes of thread pools etc. You
```

Comment:
I can't work out what "to adjust internal components like sizes of thread pools etc" means. can you rephrase this? perhaps break it into multiple sentences

Reply:
can you please make a suggestion?

Reply:
my problem is that I don't know that this is intended to mean, so am unable to make a suggestion at the moment

this setting is for the number of CPUs used. I don't understand what it has to do with the size of thread pools. the relationship needs to be explained

the way it's phrased at the moment also suggests that thread pool sizing is only one "component" affected by the number of CPUs, but it's not clear to me what else might be affected. I also think this should be explained

Reply:
Explaining what else will be affected will result in very internal technical description (schedulers, PageCacheRecycler), I'm not sure if this will help and is maybe a bit out of scope.